File: app/src/main/java/be/vergauwen/simon/androidqualitycontrol/core/service/GithubAPI.java
package be.vergauwen.simon.androidqualitycontrol.core.service;

import be.vergauwen.simon.androidqualitycontrol.core.model.GitHubRepo;
import java.util.List;
import retrofit2.http.GET;
import rx.Observable;

public interface GithubAPI {
    @GET("/users/google/repos")
    Observable<List<GitHubRepo>> getRepos();
}

File: app/src/main/kotlin/be/vergauwen/simon/androidqualitycontrol/core/service/GithubAPI.kt
package be.vergauwen.simon.androidqualitycontrol.core.service

import be.vergauwen.simon.androidqualitycontrol.core.model.GitHubRepo
import retrofit2.http.GET
import rx.Observable

interface GithubAPI {

  @GET("/users/google/repos")
  fun getRepos(): Observable<List<GitHubRepo>>
}